tests/test_callback_for.py

```diff
@@ -0,0 +1,64 @@
+import scrapy
+import pytest
+from scrapy.utils.reqser import request_to_dict, request_from_dict
+
+from scrapy_po.webpage import ItemPage, WebPage
+from scrapy_po.utils import callback_for, DummyResponse
+
+
+class FakePage(ItemPage):
+
+    def to_item(self):
+        return 'it works!'
+
+
+class MySpider(scrapy.Spider):
+
+    name = 'my_spider'
+    parse_item = callback_for(ItemPage)
+    parse_web = callback_for(WebPage)
+
+
+def test_callback_for():
+    """Simple test case to ensure it works as expected."""
+    cb = callback_for(FakePage)
+    assert callable(cb)
+
+    fake_page = FakePage()
+    response = DummyResponse('http://example.com/')
+    assert list(cb(response=response, page=fake_page)) == ['it works!']
+
+
+def test_callback_serialization():
```

Comment:
A nitpick: could you please split this test into separate tests? Probably comments provide good splitting points.